docs/upgrade_tails_usbs.rst

```diff
@@ -0,0 +1,124 @@
+
+  #. The *Admin Workstation* (allowed to connect to the Internet)
+  #. The *Journalist Workstation* (allowed to connect to the Internet)
+  #. The *Secure Viewing Station* (airgapped, never allowed to connect to any network)
+
+You are responsible for keeping your Tails USBs updated. This guide will
+explain that process.
+
+One-Click Updates
+-----------------
+
+For the Internet-connected Tails workstations (the *Admin Workstation* and
+the *Journalist Workstation*), you'll see a notification when a new version
+of Tails is available. We recommend that you :ref:`back up your existing
+configuration<Step 1>` and upgrade as soon as possible.
+
+|Upgrade Notification|
+
+It usually takes some time for updates to download, so keep that in mind when choosing when to upgrade.
+
+.. |Upgrade Notification| image:: images/tails_upgrade_notification.png
+
+
+Manual Updates
+--------------
+
+However, because the *Secure Viewing Station (SVS)* is airgapped, it cannot
+receive upgrade notifications, so it will need to be updated manually.
+
+.. important:: 
+        The *SVS* stores some of SecureDrop's most sensitive data (decrypted submissions, the SecureDrop Application private key), so we **strongly** encourage you to manually upgrade the *SVS* whenever a new version of Tails is released.
+
+The Tails project releases updates `every 6 weeks`_. Occasionally they
+release a new version ahead of the normal cycle in order to address a
+security issue. For regular Tails OS and security information, check out the
+Tails `Security page`_ and subscribe to the Tails RSS/Atom feed.
+
+.. _every 6 weeks: https://tails.boum.org/support/faq/index.en.html
+.. _Security page: https://tails.boum.org/security/index.en.html
+
+
+Tails Manual Update Process at a Glance
+----------------------------------------
+
+The process for manually updating a Tails workstation is:
+
+  #. Recommended: :ref:`Make a backup of the Tails workstation USB that you intend to upgrade<Step 1>`. For simplicity, we'll refer to the workstation you want to upgrade as the *Secure Viewing Station*, although this process would work on any Tails USB.
+  #. Install the :ref:`latest version of Tails<Step 2>` on your *primary Tails USB*.
+  #. Use the primary Tails USB to :ref:`perform a manual update of the Secure Viewing Station<Step 3>` USB on an offline (airgapped) machine.
+
+Prerequisites
+-------------
+
+You will need:
+
+  #. Your Admin Workstation computer
+  #. A *primary Tails USB* stick (you may still have one; it was used to create the Admin Workstation, Secure Viewing Station, and Journalist Workstation Tails USBs during the initial SecureDrop Install process)
+  #. Your *existing Tails USBs* that you want to update manually (*Secure   Viewing Station*, optionally *Admin Workstation* and *Journalist Workstation*)
+  #. A *backup USB stick* or other storage device to back up the data on your existing Tails USBs
+  #. Your airgapped *Secure Viewing Station* computer
```

Comment:
- Glossary terms should always be italicized (it gets a bit tedious, I know). This applies throughout the rest of the doc, where you have many uses, e.g., of Secure Viewing Station that should be italicized
- "Existing Tails USB" and "backup USB" need not be italicized.

Primary is a bit of an edge case, it's not really a term with standardized usage as you observed in #4687; until we do a scrub on that I think it's fine to treat it as a defined term in the scope of this document (i.e. italicize).